lib/index.js

```diff
@@ -472,8 +472,8 @@ Nightwatch.prototype.startSession = function () {
 
   var request = new HttpRequest(options);
   request.on('success', function(data, response, isRedirect) {
-    if (data && data.sessionId) {
-      self.sessionId = self.api.sessionId = data.sessionId;
+    if (data && data.value.sessionId) {
```

Comment:
It is true that sessionId return from firefox (selenium grid) with version 56+ is `data.value.sessionId`, however chrome (selenium grid) is still returning it as `data.sessionId`... So if this is applied, tests against firefox might work but then those agaisnt chrome would fail.

Reply:
On chrome it also return both `data.sessionId` and `data.value.sessionId`

Reply:
Faced this issue when was running nightwatch test in firefox browser inside selenoid docker container.

I've made these changes locally and it fixed the issue.

```
let sessionId = data.sessionId === undefined ? data.value.sessionId : data.sessionId
        if (data && sessionId) {
            self.sessionId = self.api.sessionId = sessionId;
```

I hope this PR will be merged as soon as possible